### DISCUSSION TITLE
add background to menu header

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [elchininet/custom-sidebar#205] [Ideas]
another functionality that would be very welcome, would be to set a background to the top menu title area (only)

consider card-mod

```
      .menu {
        background: {% if feest %} top / cover url(/local/images/balloons.png) !important {% endif %}
      }
```

and result:

<img width="271" alt="Scherm­afbeelding 2024-11-14 om 20 36 51" src="https://github.com/user-attachments/assets/088f8905-ac5b-4578-b70a-4d63fc8fa2fd">

this is more restricted than the 

```
sidebar_background: >
  [[[ @partial overzicht
      return meteoalarm ? weeralarm
      : kerst ? 'url(/local/season/kerst_smurfen.png)'
              : 'var(--sidebar-background-color)'; ]]]
```

which I also use.

Hoping you can add this and have a true header background option for your plugin, without having to resort to card-mod (and its jinja templates.....)

**Comment by elchininet** [comment]
You can do it with the `menu_background` option.

<img width="850" alt="image" src="https://github.com/user-attachments/assets/03679993-4919-4422-93fc-c689d27c37ca">

**Reply by Mariusthvdb** [reply]
Nice!
I missed that.. sorry

works with:
```
menu_background: >
  [[[ @partial familie
      return feest ? 'top / cover url(/local/images/balloons.png)' : 'none'; ]]]
```
